app/services/transaction_service/process/preauthorize.rb

```diff
@@ -173,7 +170,7 @@ def initiate_booking(tx:)
           customerId: tx[:starter_uuid],
           initialStatus: :paid,
           start: tx[:booking][:start_on],
-          end: end_adjusted
+          end: tx[:booking][:end_on]
```

Comment:
Noup, the `:end_on_exclusive` is Transaction Store internal. The store returns a Booking entity with `:start_on`, `:end_on` and `:duration`